docs/Makefile

```diff
@@ -45,7 +45,8 @@ clean:
 html:
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
 	@echo
-	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
+	@echo "Build finished. The HTML pages are in docs/$(BUILDDIR)/html."
+	rm -rf build
```

Comment:
Yep, that is what you spoke about here:
https://github.com/collective/icalendar/pull/359#issuecomment-1214150269